### PR TITLE
[ADD] hr_employee_service_length

### PR DESCRIPTION
**Pull request by saltonmassally**
Calculates the service length of employees

### CODE REVIEW

**Comment by ghost** [comment]
please add unit tests in this module.

File: hr_employee_service_length/README.rst [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+.. image:: http://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: http://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose mission is to support the collaborative development of Odoo features and promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
```
Comment (adrienpeiffer) [inline hr_employee_service_length/README.rst]: Can you modify the readme file according the template : https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

**Comment by adrienpeiffer** [comment]
Can you reference the AGPL license in your header ?

**Comment by saltonmassally** [comment]
@dufresnedavid  tests added
@adrienpeiffer  license added

Please review when you are chanced

**Comment by ghost** [comment]
:+1: when travis is green

**Comment by feketemihai** [comment]
:+1: after @lmignon comments.

File: hr_employee_service_length/README.rst [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+To install this module, you need to:
+
+* clone the branch 8.0 of the repository https://github.com/OCA/hr
+* add the path to this repository in your configuration (addons-path)
+* update the module list
+* search for "Employee Service Length" in your addons
+* install the module
+
+Configuration
+=============
```
Comment (lmignon) [inline hr_employee_service_length/README.rst:22]: You can remove this section since it's useless

File: hr_employee_service_length/README.rst [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+None Needed
+
+Usage
+=====
+
```
Comment (lmignon) [inline hr_employee_service_length/README.rst:28]: as @adrienpeiffer says, take a look to the template https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst and fill the  `Usage` section with the `Try me on runbot` link.

File: hr_employee_service_length/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+#    along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+###############################################################################
+{
+    'name': 'Employee Service Length',
+    'version': '8.0.1.0.0',
+    'license': 'AGPL-3',
+    'category': 'Human Resources',
+    'summary': "Calculates the length of an employee's time in a company",
+    'author': "Salton Massally <smassally@idtlabs.sl>, "
```
Comment (lmignon) [inline hr_employee_service_length/__openerp__.py:26]: The habit is to put here the name of the company while the author is referenced in the README file

**Comment by feketemihai** [comment]
@tarzan0820 The same as #161 , these modules should be merged in hr_employee_seniority...
:-1:

**Pull request closed without merging**